website/docs/cloud-docs/agents/request-forwarding.mdx

```diff
@@ -24,7 +24,8 @@ Agents register with HCP Terraform via outbound-only connections. HCP Terraform
 holds these connections open and uses them to transmit requests and responses 
 between the target API and HCP Terraform.
 
-![Request forwarding architecture diagram](/img/request-forwarding-arch.png)
+![Request forwarding architecture diagram](/img/request-forwarding-arch-light.png#light-theme-only)
```

Comment:
@tsu019 I think you forgot to include the updated file in this PR? Let me know if you need any help uploading it!

Reply:
Oh apologies, sorry I didn't realize you had another PR up for the images! Approved and merged that one now, so I'm going to rebuild the preview!